PyPoE/cli/exporter/wiki/parsers/item.py

```diff
@@ -3995,21 +4003,26 @@ def export_map(self, parsed_args):
         self.rr['AtlasNode.dat'].build_index('MapsKey')
         names = set(parsed_args.name)
         map_series_tiers = {}
+        # For each map, save off the atlas node
         for row in self.rr['MapSeriesTiers.dat']:
             maps = row['MapsKey']
+
+            # Try to find the atlas node for the map. Save that as atlas_node by breaking.
             for atlas_node in self.rr['AtlasNode.dat'].index['MapsKey'][maps]:
                 # This excludes the unique maps
-                if atlas_node['ItemVisualIdentityKey'][
-                        'IsAtlasOfWorldsMapIcon']:
+                if atlas_node['ItemVisualIdentityKey']['IsAtlasOfWorldsMapIcon']:
                     break
+            # If we couldn't find the atlas node, set atlas_node to None to clear the last value.
             else:
                 # Maps that are no longer on the atlas such as guardian maps
                 # or harbinger
                 atlas_node = None
-            if names and maps['BaseItemTypesKey']['Name'] in names or \
-                    not names:
+            
+            # Save off the atlas_node for each map in the series
+            if (names and maps['BaseItemTypesKey']['Name'] in names) or not names:
```

Comment:
If names is not None or not False, and maps[...]['Name'] key is in Names - assign `atlas_node`, but also if names IS None or False assign `atlas_node`?

Am I missing something not entirely obvious here?

Reply:
This took me a minute to understand. `names` is the set of map names to export from the command argument.
So it's either doing no filtering if it wasn't provided, or filtering down to the provided names.
I've added a comment to better explain this.